pkg/workloadmeta/collectors/internal/containerd/event_builder.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/containerd/containerd"
 	"github.com/containerd/containerd/api/events"
 	containerdevents "github.com/containerd/containerd/events"
-	"github.com/gogo/protobuf/proto"
+	"google.golang.org/protobuf/proto" // can't use gogo proto until https://github.com/gogo/protobuf/issues/678 is fixed
```

Comment:
do we need this comment? It seems that it is not plan to use back to gogo proto.

Reply:
Indeed, I removed it